src/main/java/com/hazelcast/azure/AzureDiscoveryStrategy.java

```diff
@@ -27,20 +27,13 @@
 import com.hazelcast.spi.partitiongroup.PartitionGroupMetaData;
 import com.microsoft.azure.CloudException;
 import com.microsoft.azure.PagedList;
-import com.microsoft.azure.management.compute.InstanceViewStatus;
-import com.microsoft.azure.management.compute.VirtualMachine;
-import com.microsoft.azure.management.compute.VirtualMachineInstanceView;
+import com.microsoft.azure.management.compute.*;
```

Comment:
It doesn't feel like wildcard imports are a thing in this code base. Maybe keep as is?

Reply:
Good point, I will keep it.